project/Dependencies.scala

```diff
@@ -130,7 +132,7 @@ object Dependencies {
   // TODO check if `l ++=` everywhere expensive?
   val l = libraryDependencies
 
-  val actor = l ++= Seq(config, java8Compat.value)
+  val actor = l ++= Seq(config, java8Compat.value, javaAffinity)
```

Comment:
we can try it out and continue development like this until we decide where to place it